tlsfuzzer/extract.py

```diff
@@ -322,7 +322,7 @@ def _parse_pcap(self):
                             self.server_msgs.append(timestamp)
                         else:
                             if tcp_pkt.ack != exp_clnt_ack:
-                                print("Mismatched syn/ack seq at {0}"
+                                print("Mismatched syn/ack seq at {0}\n"
```

Comment:
nit: would be nice to squash these changes into the previous commit instead having them in this unrelated one.

Reply:
good point, fixed